Site/ASAB Maestro/Descriptors/asab-remote-control.yaml

```diff
@@ -16,7 +16,7 @@ descriptor:
 
 asab:
   configname: conf/asab-remote-control.conf
-  config: {}
+  config: &config {}  # specify default configuration of the service
```

Comment:
Will this go away?
Please add a TODO at least.

Reply:
aha